go/vt/vtgate/vschema_manager.go

```diff
@@ -231,6 +237,49 @@ func (vm *VSchemaManager) updateFromSchema(vschema *vindexes.VSchema) {
 	}
 }
 
+type tableCol struct {
+	tableName string
+	colNames  sqlparser.Columns
+}
+
+var tableColHash = func(tc tableCol) string {
+	res := tc.tableName
+	for _, colName := range tc.colNames {
+		res += "|" + colName.Lowered()
+	}
+	return res
+}
```

Comment:
This hashing function works, but I am not sure if this is 100% correct... The graph package I am using for cycle detection requires this hashing function which it uses to uniquely identlify nodes. The problem is that if we have table t1 with cols `a`, `b` and `a|b`, then the hash for t1 with `a` and `b` would be `t1|a|b` and so would be the hash for `t1` with `a|b`. 
Essentially this function isn't correct if | is present in column names of tables.

Reply:
Looking for advice to fix this if possible. Essentially asking for a way to consistantly hash a list of strings without placing any constraints on them

Reply:
can add like `keyspace#table#columns`

Reply:
I talked to Andres, and he suggested we use the `sqlparser.String` method to print the columns. So now if a column has the `|` character, then it would get backticked, and eveyrthing is good now.